easybuild/easyconfigs/l/libcerf/libcerf-2.3-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'libcerf'
+version = '2.3'
+
+homepage = 'https://jugit.fz-juelich.de/mlz/libcerf'
+
+description = """
+ libcerf is a self-contained numeric library that provides an efficient and
+ accurate implementation of complex error functions, along with Dawson,
+ Faddeeva, and Voigt functions.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v%(version)s/']
+sources = ['libcerf-v%(version)s.tar.gz']
+checksums = ['cceefee46e84ce88d075103390b4f9d04c34e4bc3b96d733292c36836d4f7065']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.26.3'),
+    ('Perl', '5.36.1', '-minimal'),  # required for pod2html
```

Comment:
Since the `GCCcore-12.3.0` toolchain generation `Perl` is no longer a fat build, but split into a minimal `Perl` and `Perl-bundle-CPAN`. So I think we can use `Perl` instead of `Perl-minimal` here. I have opened a follow-up PR: https://github.com/easybuilders/easybuild-easyconfigs/pull/18565